BaseFilter/sections/general_extensions.txt

```diff
@@ -3519,9 +3519,7 @@ uploading.site#%#var o=window.open;window.open=function(a){a.indexOf("clicksor.n
 ! https://github.com/AdguardTeam/AdguardFilters/issues/3364
 vg247.com#%#AG_onLoad(function() { window.yafaIt = function() {}; });
 ! https://forum.adguard.com/index.php?threads/15364/
-deadline.com#%#Object.defineProperty(window, 'AdLoader', { get: function() { return; } });
-deadline.com#%#Object.defineProperty(window, 'pmc_admanager', { get: function() { return; } });
-deadline.com#%#Object.defineProperty(window, '_sp_', { get: function() { return; } });
+deadline.com#%#//scriptlet('set-constant', 'pmc_admanager', 'undefined')
```

Comment:
Hi @piquark6046 
Why we need this rule?
I don't see any ads without  these 3 JS-rules.

Reply:
I have the same, looks like it's not needed.

Reply:
I'll delete it by myself, I hope you don't mind.